projects/core/src/auth/store/effects/user-token.effect.ts

```diff
@@ -44,7 +42,6 @@ export class UserTokenEffects {
     switchMap(({ userId, refreshToken }) => {
       return this.userTokenService.refreshToken(refreshToken).pipe(
         map((token: UserToken) => {
-          token.userId = userId;
           const date = new Date();
           date.setSeconds(date.getSeconds() + token.expires_in);
           token.userId = userId;
```

Comment:
Why doesn't the refresh set the userId to current also?

Reply:
Because as far as I can see from the code, the refresh token copies over the userId from the expired token.  Does this look accurate?

Reply:
Really, I thought the `this.userTokenService.refreshToken(refreshToken)` calls returns a new function

Reply:
I applied the improvement we discussed: remove userId from the RefreshUserToken Action to remove ambiguity about what to expect from the token refresh.